src/commons/api/ca.rs

```diff
@@ -406,20 +406,27 @@ impl Eq for RcvdCert {}
 
 #[derive(Clone, Debug, Deserialize, Eq, PartialEq, Serialize)]
 pub struct TrustAnchorLocator {
-    uris: Vec<uri::Https>, // We won't create TALs with rsync, this is not for parsing.
+    uris: Vec<uri::Https>,
```

Comment:
Might be good to rename references to `uris` to `http_uris` now that there is also `rsync_uri`?

Reply:
I didn't because it will break deserialization. I could have an alias for serde, but I figured it wasn't important enough. Can be persuaded.. if you prefer the alias..